src/js/common/schemaform/definitions/applicantInformation.js

```diff
@@ -1,10 +1,10 @@
 import _ from 'lodash/fp';
 
-import currentOrPastDateUI from '../../common/schemaform/definitions/currentOrPastDate';
-import fullNameUI from '../../common/schemaform/definitions/fullName';
+import currentOrPastDateUI from './currentOrPastDate';
```

Comment:
Let's make a `pages` folder, instead of putting this in `definitions`. We've been using that separation up until now and I think it's useful context.